src/roktManager.ts

```diff
@@ -142,6 +163,21 @@ export default class RoktManager {
         this.processMessageQueue();
     }
 
+    public processEvent(event: SDKEvent): void {
+        const hashedEvent = KitFilterHelper.hashEventMessage(
+            event.EventName,
+            event.EventCategory as valueof<typeof EventType>,
+            event.EventDataType as valueof<typeof MessageType>
+        ).toString();
+
+        if (Object.keys(this.placementEventMappingLookup).includes(hashedEvent)) {
+            const mappedValue = this.placementEventMappingLookup[hashedEvent];
+            this.store.setSessionSelectionAttributes({
```

Comment:
If `this.placementEventMappingLookup` is an object with keys of `hashedEvent`, why turn it into an array to look up the key of `hashedEvent`?


```suggestion
        if (this.placementEventMappingLookup[hashedEvent]) {
```